src/main/features/response/guards/countyCourtJudgmentRequestedGuard.ts

```diff
@@ -0,0 +1,17 @@
+import * as express from 'express'
+import { Paths as DashboardPaths } from 'dashboard/paths'
+import User from 'idam/user'
+const logger = require('@hmcts/nodejs-logging').getLogger('response/guards/countyCourtJudgmentRequestedGuard')
+
+export default class CountyCourtJudgmentRequestedGuard {
```

Comment:
Please change it to not be a default export.

Reply:
ok